src/client/coll/results.rs

```diff
@@ -0,0 +1,104 @@
+use bson;
+use bson::Bson;
+
+use std::collections::BTreeMap;
+
+/// Results for a bulk write operation.
+#[derive(Clone)]
+pub struct BulkWriteResult {
```

Comment:
Possibly `Outcome`, although I'm wary of moving away from the naming conventions in the mongo driver spec.